### DISCUSSION TITLE
Code Scanning no ability to filter alerts

### DISCUSSION THREAD

**Discussion by dibbdob** [community/community#23652] [New to GitHub]
<p>We’ve just run a code scan and need to filter on the large set of results that are returned.  However, when we go to the Code Scanning page to view the results, selecting Rule or Tag no entries are offered.</p>
<p><img src="https://user-images.githubusercontent.com/6143665/181078647-a74c68ea-a826-4412-bb6f-fd73e8297f20.png" alt="image" data-base62-sha1="i5N2e0Ots6RKPTMmAuxqiGNDiJt" width="613" height="274"></p>
<p>We do know that the filter has worked in the past (27/10/2020)</p>
<p>Any ideas or thoughts to fix this?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/dibbdob">@dibbdob</a> :wave:</p>
<p>We are aware of this issue and a resolution is being worked on. Thanks for your patience.</p>

**Comment by dibbdob** [comment]
<p>Great, the problem appears to have been fixed and filtering on tags is now available!</p>
